scripts/src/main/resources/scripts/command/docker

```diff
@@ -38,55 +38,51 @@ function doResolveDownloadUrl() {
 function doSetup() {  
   if [ "${DOCKER_EDITION}" != "DOCKERDESKTOP" ]
   then
-    echo "Rancher Desktop set as option"
     if uname -r | grep -i microsoft
     then 
       doFail "It seems that you are running the setup from inside a WSL shell.\nPlease use git-bash for that purpose. Otherwise your OS will be interpreted as Linux"
     else
-      if doIsRancherInstalled
+      if doIsDockerInstalled
```

Comment:
actually this check should be done independent of `${DOCKER_EDITION}`.
If `docker` is already present, `doSetup` has nothing left to do.